components/infobox/wikis/leagueoflegends/infobox_unit_champion.lua

```diff
@@ -0,0 +1,213 @@
+		Cell { name = 'Attack Damage', content = { bonusPerLevel(_args.damage, _args.damagelvl) } },
+		Cell { name = 'Attack Speed', content = { bonusPerLevel(_args.attackspeed, _args.attackspeedlvl) } },
+		Cell { name = 'Attack Range', content = { _args.attackrange } },
+		Cell { name = 'Armor', content = { bonusPerLevel(_args.armor, _args.armorlvl) } },
+		Cell { name = 'Magic Resistance', content = { bonusPerLevel(_args.magicresistance, _args.magicresistancelvl) } },
+		Cell { name = 'Movement Speed', content = { _args.movespeed } },
+		Title { name = 'Esports Statistics' }
+	)
+
+	local stats = ChampionWL.create({ champion = _args.championname or _pagename })
+	stats = mw.text.split(stats, ';')
+	local winPercentage = (tonumber(stats[1]) or 0) / ((tonumber(stats[1]) or 0) + (tonumber(stats[2]) or 1))
+	winPercentage = Math.round(winPercentage, 4) * 100
+	local statsDisplay = (stats[1] or 0) .. 'W : ' .. (stats[2] or 0) .. 'L (' .. winPercentage .. '%)'
+	table.insert(widgets, Cell { name = 'Win Rate', content = { statsDisplay } })
+
+	return widgets
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	if id == 'header' then
+		return {
+			Header {
+				name = _args.championname,
+				subHeader = _args.title,
+				image = _args.image,
+				imageDefault = _args.default,
+				imageDark = _args.imagedark or _args.imagedarkmode,
+				imageDefaultDark = _args.defaultdark or _args.defaultdarkmode,
+			},
+		}
+	elseif id == 'caption' then
+		table.insert(widgets, Center { content = { _args.quote } })
+	elseif id == 'type' then
+		local breakDownContents = {}
+		local region = _args.region
+		if not String.isEmpty(region) then
+			region = '<b>Region</b><br>' .. Template.safeExpand(_frame, 'Region icon', { region }, '')
+			table.insert(breakDownContents, region)
+		end
+		local primaryRole = _args.primaryrole
+		if not String.isEmpty(primaryRole) then
+			primaryRole = '<b>Primary Role</b><br>' .. Template.safeExpand(_frame, 'Class icon', { primaryRole }, '')
+			table.insert(breakDownContents, primaryRole)
+		end
+		local secondaryRole = _args.secondaryrole
+		if not String.isEmpty(secondaryRole) then
+			secondaryRole = '<b>Secondary Role</b><br>' .. Template.safeExpand(_frame, 'Class icon', { secondaryRole },
+				'')
+			table.insert(breakDownContents, secondaryRole)
+		end
+		return {
+			Breakdown { classes = { 'infobox-center' }, content = breakDownContents },
+			Cell { name = 'Real Name', content = { _args.realname } },
+		}
+	elseif id == 'cost' then
+		local cost = ''
+		if not String.isEmpty(_args.costbe) then
+			cost = cost .. _args.costbe .. ' ' .. BLUE_ESSENCE_ICON
+		end
+		if not String.isEmpty(_args.costrp) then
+			if cost ~= '' then
+				cost = cost .. '&emsp;&ensp;'
+			end
+			cost = cost .. _args.costrp .. ' ' .. RIOT_POINTS_ICON
+		end
+		return {
+			Cell { name = 'Price', content = { cost } },
+		}
+	end
```

Comment:
```suggestion
	elseif id == 'custom' then
		self.caller:getCustomCells(widgets)
	end
```

only because that is a pretty big chunck, if it were a small one we would just move it in here